dev.aunetx.deezer.yml

```diff
@@ -0,0 +1,62 @@
+app-id: dev.aunetx.deezer
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+tags: ["proprietary"]
+finish-args:
+  # Basic permissions
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it really support wayland? if yes then like 15 is `fallback-x11`. otherwise remove this.

Reply:
It supported native wayland, but support was lost (https://github.com/aunetx/deezer-linux/issues/12) and I forgot about it. I remove this :)

Reply:
Am using it on wayland (Fedora 38 WS + Wayland + GNOME) as flatpak without any issue